packages/bpk-docs/src/pages/BadgePage/BadgePage.js

```diff
@@ -43,10 +43,12 @@ const components = [
       </Paragraph>,
     ],
     examples: [
-      <BpkBadge className={badgeClassName}>Apples</BpkBadge>,
-      <BpkBadge className={badgeClassName}>Bananas</BpkBadge>,
-      <BpkBadge className={badgeClassName}>Strawberries</BpkBadge>,
-      <BpkBadge className={badgeClassName}>Pears</BpkBadge>,
+      <div className={getClassName('bpkdocs-badge-page__badge-layout-dark')}>
```

Comment:
Yeah it's basically because `BadgeLayout` comes with it's own padding. If we were able to set a dark option on the docs page we could use the same mecahnism for both.